cluster/gce/gci/configure-helper.sh

```diff
@@ -95,7 +95,11 @@ function main() {
     gke-master-start
   else
     create-node-pki
-    create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
+    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
+      create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
+    else
+      create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME} "8080" "http"
+    fi
```

Comment:
ENABLE_APISERVER_INSECURE_PORT not only used for tenant/resource partition. if use this as condition, may impact regular cluster logic.

Reply:
as we talked in the meeting, if this is only for POC, that's ok.